src/generated/resources/data/supplementaries/tags/items/throwable_bricks.json

```diff
@@ -0,0 +1,6 @@
+{
+  "replace": false,
+  "values": [
+ "ceramics:porcelain_brick"
```

Comment:
`src/generated` contains stuff that is datagenned. Anything in this folder that did not come from a data generator will automatically be deleted next time I run datagen.

Please update the data generator if you wish to make this change.